Doc/whatsnew/3.14.rst

```diff
@@ -141,6 +141,12 @@ pathlib
 
   (Contributed by Barney Gale in :gh:`73991`.)
 
+pickle
+------
+
+* Set the default protocol version on the :mod:`pickle` module to 5.
+  For more details, please see :ref:`pickle protocols <pickle-protocols>`.
+
```

Comment:
Shouldn't this be after `pdb`? (assuming alphabetical order)

Reply:
Indeed, thanks :) Will update.